src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/ExplainIT.java

```diff
@@ -211,4 +212,19 @@ public void testContentTypeOfExplainRequestShouldBeJson() throws IOException {
 
         assertEquals("application/json; charset=UTF-8", response.getHeader("content-type"));
     }
+
+    @Test
+    public void testExplainRequestPrettyFormatted() throws IOException {
+
+        String expectedOutputFilePath = TestUtils.getResourceFilePath(
+                "src/test/resources/expectedOutput/explain_format_pretty.json");
+        String expectedOutput = Files.toString(new File(expectedOutputFilePath), StandardCharsets.UTF_8);
+
+        String query = "SELECT firstname FROM " + TEST_INDEX_ACCOUNT;
+        String explain = explainQuery(query);
+        String result = new JsonPrettyFormatter().format(explain);
+
+        Assert.assertThat(result.replaceAll("\\s", "").replaceAll("\\n", "*"),
```

Comment:
Why we need all those replaceAll, if we read file and do translation? Why can't we just do 

    assertThat(result, isEqual(expectedOutput))?